osquery/distributed/distributed.cpp

```diff
@@ -17,10 +17,17 @@
 #include <osquery/sql.h>
 #include <osquery/system.h>
 
+#include <rapidjson/document.h>
+#include <rapidjson/error/en.h>
+#include <rapidjson/stringbuffer.h>
+#include <rapidjson/writer.h>
+
 #include "osquery/core/conversions.h"
 #include "osquery/core/json.h"
 
-namespace pt = boost::property_tree;
+#undef GetObject
```

Comment:
Yes because RapidJSON defines GetObject which is defined globally on Windows causing compilation errors.